src/rulesets/properties.ts

```diff
@@ -96,6 +96,7 @@ export const rules = {
     bodyProperties,
   }: SnykApiCheckDsl) => {
     bodyProperties.added.must("not be required", (property, context, docs) => {
+      if (context.bodyAdded) return;
```

Comment:
for rules about required, they can be required only if the body was also added at the same time (there to start).